site/tutorials/tutorial-two-spring-amqp.md

```diff
@@ -384,18 +384,20 @@ set to 0 the behavior would be round robin messaging as described above.
   </div>
 </div>
 
-However, with the prefetchCount set to 250 by default,
+However, with the `prefetchCount` set to 250 by default,
 this tells RabbitMQ not to give more than 250 messages to a worker
 at a time. Or, in other words, don't dispatch a new message to a
 worker while the number of unacked messages is 250.
 Instead, it will dispatch it to the next worker that is not still busy.
 
+Still, `prefetchCount` can be changed via 
```

Comment:
I'd drop the "still", add a note that the value of 1 is very inefficient for most workloads and link to `/confirms.html#channel-qos-prefetch` where the reader can learn more.

Reply:
Thanks for the feedback, I have done adjustments.